mysql-test/suite/tokudb.bugs/r/alter_external_lock_assert.result

```diff
@@ -18,4 +16,6 @@ ALTER TABLE t1_tokudb_tokudb_smallvc_250 MODIFY c1 DATE;
 UNLOCK TABLES;
 COMMIT;
 DROP TABLE IF EXISTS i,p,t1_tokudb_tokudb_smallvc_250;
+Warnings:
```

Comment:
Don't drop "p"

Reply:
Ahh, actually, that seems to be the cause of the tests need for the duplicate DROP.